composer.json

```diff
@@ -16,8 +16,9 @@
     }
   ],
   "require": {
-    "php": ">=5.3",
-    "shopgate/cart-integration-sdk" : "~2.9.68"
+    "php": ">=5.4",
```

Comment:
just run `php-cs-fixer fix --config=.php-cs.dist` in the root & commit. That will fix all the travis issues with styling. You can also revert to 5.3 support again since it will adjust all your shorthand array entries. We want to stay compatible on the old framework side that does not utilize you shopware 5.2 version code. 
At least I believe it is still possible to use the old code when below shopware 5.2, right?

Reply:
Year the plugin is written for the old plugin system from shopware so it also works below 5.2

Reply:
alrighty, then run the fixer to revert the array shorthand syntax + other minor things and we shall be good to go with this PR round. You'll need to download the `php-cs-fixer` if you don't have it though. `brew install php-cs-fixer` should do it if on mac.